controller/identity/validator.go

```diff
@@ -0,0 +1,97 @@
+package identity
+
+import (
+	"context"
+	"fmt"
+	"strings"
+
+	kauthnApi "k8s.io/api/authentication/v1"
+	kauthzApi "k8s.io/api/authorization/v1"
+	k8s "k8s.io/client-go/kubernetes"
+	kauthn "k8s.io/client-go/kubernetes/typed/authentication/v1"
+	kauthz "k8s.io/client-go/kubernetes/typed/authorization/v1"
+
```

Comment:
No extra newline in imports

Reply:
The intent was to group the repo-local packages separately from external deps.

Reply:
I wonder if this is something we can enforce with a lint...